apps/compilersforhumans/src/styles/globals.css

```diff
@@ -34,6 +34,37 @@
 .text-stroke-white {
 }
 
+/* Fix Autocomplete styles for color themes */
+
+@media (prefers-color-scheme: dark) {
+  input:-webkit-autofill,
```

Comment:
This fixes autofill contrast issues reported in #77. Tested in Chrome and Safari on both desktop and mobile. Couldn't test on Android device, but looks like it was the same issue.